geofront/backends/cloud.py

```diff
@@ -94,6 +107,10 @@ class CloudRemoteSet(collections.abc.Mapping):
     .. _Libcloud: http://libcloud.apache.org/
     __ https://libcloud.readthedocs.org/en/latest/compute/
 
+    .. versionchanged:: 0.5.0
```

Comment:
I'd like to include this patch in 0.4.1 release.  Could you change this to 0.4.1?